core/src/main/java/org/mobicents/ha/javax/sip/LoadBalancerHeartBeatingServiceImpl.java

```diff
@@ -639,7 +639,7 @@ protected void updateConnectorsAsSIPNode() {
 						if(jvmRoute != null) node.getProperties().put("jvmRoute", jvmRoute);
 						
 						node.getProperties().put("version", System.getProperty("org.mobicents.server.version", "0"));
-						
+						node.getProperties().put("sessionId", ""+System.currentTimeMillis());
 						if(gracefullyShuttingDown) {
```

Comment:
let's do it properly to avoid any issues if connectors are modified even rarely, we will have a hard time debugging it.